### ISSUE TITLE
Cannot use with node@13 and node@14

### ISSUE THREAD

**Issue by aalexgabi** [xeroxinteractive/browserslist-browserstack#516]
```
% yarn add -D browserslist-browserstack
yarn add v1.21.1
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
error browserslist-browserstack@3.0.2: The engine "node" is incompatible with this module. Expected version ">=10 <=12". Got "13.7.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by aalexgabi** [comment]
@AndrewLeedham Why not blacklist versions instead of whitelisting?
```
  "engines": {
    "node": ">=10 <=14"
  },
```
The module won't work with any new version of Node or any version of Node that is in development without doing a new commit right? I don't think this is a good practice because:
- there needs to be an update to the code for every new major version of Node
- before the update is done users will not be able to use new versions of Node, slowing down the Node ecosystem updates
- if this module is not maintained one day it won't be usable without a fork anymore.

Until an incompatibility is not proven it's not there. Most new major versions of Node do not break libraries.

**Comment by AndrewLeedham** [comment]
Hi @aalexgabi, that is a fair point. Our current strategy is to scope it to known working versions. That has come from our internal projects. Perhaps we need to adopt the strategy you are suggesting for our OSS projects 👍 

Happy to review a PR if you need this urgently. Otherwise, I will get to it when I have some spare time.

**Comment by AndrewLeedham** [comment]
@aalexgabi I changed the engines range to `>=10` thanks for the suggestion. Released in v3.1.1 🚀

**Comment by aalexgabi** [comment]
@AndrewLeedham Thank you!
